scripts/docker/dockerfile_clang-10

```diff
@@ -1,28 +1,26 @@
-FROM axom/compilers:ubuntu-20-clang-10_07-19-21_23h-07m
+FROM axom/compilers:ubuntu-20-clang-10_08-24-21_07h-18m
 LABEL maintainer="bramwell1@llnl.gov"
 ARG branch=develop
 
 SHELL ["/bin/bash", "-c"]
 RUN sudo apt-get update -y
-RUN sudo apt-get install ssh libopenblas-dev mpich gfortran -fy
+RUN sudo apt-get install doxygen gfortran graphviz libopenblas-dev mpich python3-sphinx ssh texlive-full -fy
```

Comment:
These changes should be propagated to `dockerfile.in` - not sure if anyone else uses that workflow (`build_new_dockerfile.sh`) for creating new images but for what it's worth I used it to create the GCC 9 image

Reply:
done!  That file was very out of date.....

Reply:
Yeah, I don't even know the last time we used that directly...